Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
```

Comment:
@qodesmith Ok, I removed the blank spaces. I wasn't sure what common coding etiquette was needed here. So as long as my indentation is uniform, I don't really need to put the extra spaces in between lines? I was doing that for visibility but I guess it's kind redundant.